server/app.ts

```diff
@@ -32,14 +33,18 @@ export default function createApp(services: Services): express.Application {
   app.use(setUpWebSession())
   app.use(setUpWebRequestParsing())
   app.use(setUpStaticResources())
-  nunjucksSetup(app, services.applicationInfo)
+  nunjucksSetup(app)
   app.use(setUpAuthentication())
   app.use(authorisationMiddleware())
   app.use(setUpCsrf())
   app.use(setUpCurrentUser(services))
+  app.use(populateCurrentUserRoles())
 
-  app.use(routes(services))
+  app.use(routes(standardRouter(services.userService)))
 
+  // authenticated by passport / HMPPS Auth
+  app.use('/', setUpAuthentication())
+  app.use('/', authorisationMiddleware(['ROLE_SLM_SCAN_BARCODE', 'ROLE_SLM_ADMIN']))
```

Comment:
Not needed? Authentication and authorisation already set up in lines 37-38 (without specifying a path so defaults to `/`)

Reply:
Updated. This file is back to the template version.